### ISSUE TITLE
Additional copy button for code URL

### ISSUE THREAD

**Issue by VlntnLnhrdt** [ls1intum/Artemis#9753]
### Is your feature request related to a problem?

As a user / student: When you start a new exercise, you must then click on "Code </>" and then on "Copy URL" on the far left of the pop up to copy the URL. It's not a big problem, but if you have multiple smaller task and are working on a laptop with a touchpad, its a litte bit annoying to do that every time.

### Describe the solution you'd like

An additional button attached to the existing "Code </>" button with a copy function would be very helpful, as shown in my mockup down below.
![Artemis-copyCode Feedback](https://github.com/user-attachments/assets/20a00b7e-8a1d-41b5-a5af-7912687956c1)

### Describe alternatives you've considered

Another option would be to copy the URL as soon as you click on "Start exercise".
Although this overwrites the previous clipboard with the URL, I believe that when you start a new task, you rarely need the clipboard with the old copied content again.

### Additional context

Of course, I don't know how often the other options "Open in VS Code" or "Open in Git client" are used and whether the way via "Copy URL" is the main use case at all, so it may also be that "my" way is rather an isolated case and this new feature would only make the interface more complicated / overloaded.

**Comment by hanna20022005** [comment]
yeah, a separate button would be nice as rn you still have to click on the "code" button first, having a dedicated button would save some time in the long run

<img width="1185" height="195" alt="Image" src="https://github.com/user-attachments/assets/5c79711c-471f-4ec4-b2f1-53fc5a04adbf" />

**Comment by Jess-hub09** [comment]
I agree with this suggestion, copying the repository URL usually happens right after starting an exercise, and the extra clicks through the popup can feel a bit tedious, especially on smaller laptops. A small “copy” button directly next to the Code </> button would streamline the workflow without adding much UI clutter. Automatically copying the URL on “Start exercise” could also work, but having an explicit quick-access copy button seems like the most user-friendly and least intrusive option. Therefore this is still relevant.

<img width="1108" height="150" alt="Image" src="https://github.com/user-attachments/assets/829139ce-001a-4071-aff5-55b105e9c09c" />

**Comment by ilgintaseli** [comment]
I agree with the suggestion and such feature has not been implemented yet.
<img width="1055" height="234" alt="Image" src="https://github.com/user-attachments/assets/b4d5c768-bc71-46a9-8c00-f206f156e182" />
